release/k3s/k3s.go

```diff
@@ -274,7 +274,7 @@ func rebaseAndTag(r *ecmConfig.K3sRelease, u *ecmConfig.User) ([]string, error)
 // Finally, the function returns the publicKeys variable, which is now an ssh.AuthMethod, and a nil error.
 func getAuth(privateKey string) (ssh.AuthMethod, error) {
 	if privateKey == "" {
-		privateKey = fmt.Sprintf("%s/.ssh/id_rsa", os.Getenv("HOME"))
+		privateKey = os.Getenv("HOME") + "/.ssh/id_rsa"
```

Comment:
I have had problems with private keys before. 
If no private key is provided, we are assuming that there will be a private key in the default directory with the default name. 

I don't have the entire context here, but it seems reasonable to check if this file exists before accessing it.

Reply:
You're not wrong. I can address that in another PR.